reactapp/src/components/header.js

```diff
@@ -18,7 +18,7 @@ class Header extends React.Component {
             <nav className="navbar navbar-expand-lg navbar-dark teal mb-4 pageNavHeader" id="home">
                 <span className="navbar-brand" href="#">
                     <Link to="/">
-                        <img className="logoImg" alt="ClimateChange.AI" src={logo}/>
+                        <img className="logoImg" alt="ClimateChange.AI" src={logo} onClick={()=>{window.location.reload()}}/>
```

Comment:
On clicking the logo, the page is reloaded (We are on a single page app - So reload automatically resets to the original "home" state.